mindmeld/components/_util.py

```diff
@@ -38,3 +39,207 @@ def _get_module_or_attr(module_name: str, func_name: str = None):
     if func_name not in dir(m):
         raise ImportError(f"Cannot import {func_name} from {module_name}")
     return getattr(m, func_name)
+
+
+class TreeNode:
+    def __init__(self, value: str, parent=None, children=None, allow=None):
```

Comment:
It's not immediately clear that `value` refers to the name of node, can be confused with referring to the child nodes. Would be helpful to add some documentation clarifying this or consider a different attribute name such as `node_name` or `node_label`

Reply:
done